apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -156,12 +161,61 @@ export default class SetupChecklist extends Component {
     this.detect();
   }
 
-  render() {
-    const surveyLink = (
+  renderPlatformSpecificSteps() {
+    if (isCodeOrgBrowser()) {
+      // Maker Toolkit Standalone App
+      return (
+        <ValidationStep
+          stepName="Code.org Browser"
+          stepStatus={this.state[STATUS_SUPPORTED_BROWSER]}
+        />
+      );
+    } else if (isChromeOS() || isChrome()) {
+      // Chromebooks - Chrome App
+      return (
+        <ValidationStep
+          stepName={'Chrome App installed' + (isChromeOS() ? '' : ' (Legacy)')}
+          stepStatus={this.state[STATUS_APP_INSTALLED]}
+        >
+          Please install the
+          {' '}
+          <a
+            href="https://chrome.google.com/webstore/detail/codeorg-serial-connector/ncmmhcpckfejllekofcacodljhdhibkg"
+            target="_blank"
+          >
+            Code.org Serial Connector Chrome App
+          </a>.
+          <br/>Once it is installed, come back to this page and click the
+          "re-detect" button, above.
+          <br/>If a prompt asking for permission for Code Studio to connect
+          to the Chrome App pops up, click Accept.
+          {this.surveyLink()}
+        </ValidationStep>
+      );
+    } else {
+      // Unsupported Browser
+      return (
+        <ValidationStep
+          stepName="Using a supported browser"
+          stepStatus={Status.FAILED}
+        >
+          {isChrome() && `It looks like your Chrome version is ${getChromeVersion()}.`}
```

Comment:
Is this necessary? Seems like this could never happen based on the if above

Reply:
Good point.  I'll tear this out.